src/main/java/com/clickhouse/kafka/connect/sink/ClickHouseSinkConfig.java

```diff
@@ -245,6 +248,17 @@ public ClickHouseSinkConfig(Map<String, String> props) {
         this.dbTopicSplitChar = props.getOrDefault(DB_TOPIC_SPLIT_CHAR, "");
         this.keeperOnCluster = props.getOrDefault(KEEPER_ON_CLUSTER, "");
         this.bypassRowBinary = Boolean.parseBoolean(props.getOrDefault("bypassRowBinary", "false"));
+        this.dateTimeFormats = new HashMap<>();
+        String dateTimeFormatsString = props.getOrDefault(DATE_TIME_FORMAT, "").trim();
+        if (!dateTimeFormatsString.isBlank()) {
+            String [] stringSplit = dateTimeFormatsString.split(";");
```

Comment:
Using `;` as a separator might be fine, but I was thinking more as `"xxx={format1}","yyy={format2}","zzz={format3}"`
Split `","` truncate `"` and to parse it inside i think it is more error-prone